### PR TITLE
Ruleset Engine: Update API examples and placeholders

### PR DESCRIPTION
**Pull request by pedrosousa**
* Replace Email + API key auth with API token auth.
* Remove unnecessary `-X GET` and `-X POST` command-line parameters (for `curl`).
* Update placeholder formatting.
* Fix some minor errors in the API examples.

### CODE REVIEW

File: products/ruleset-engine/src/content/basic-operations/deploy-rulesets.md [hunk 1]
```diff
@@ -52,24 +51,24 @@ header: Response
 ---
 {
   "result": {
-    "id": "{zone-level-phase-ruleset-id}",
+    "id": "<ZONE_PHASE_RULESET_ID>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/basic-operations/deploy-rulesets.md:54]: Updated placeholder.

File: products/ruleset-engine/src/content/basic-operations/deploy-rulesets.md [hunk 2]
```diff
@@ -52,24 +51,24 @@ header: Response
...
         "action_parameters": {
-          "id": "{cloudflare-managed-ruleset-id}",
+          "id": "<CLOUDFLARE_MANAGED_RULESET_ID>",
           "version": "3"
         },
         "expression": "true",
         "description": "Execute Cloudflare Managed Ruleset on my zone ruleset",
         "last_updated": "2021-03-18T18:08:14.003361Z",
-        "ref": "{ruleset-ref}",
+        "ref": "<RULE_REF>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/basic-operations/deploy-rulesets.md:71]: Fixed placeholder.

File: products/ruleset-engine/src/content/basic-operations/view-rulesets.md [hunk 3]
```diff
@@ -37,7 +35,7 @@ header: Response
 {
   "result": [
     {
-      "id": "{zone-level-phase-ruleset-id}",
+      "id": "<ZONE_PHASE_RULESET_ID>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/basic-operations/view-rulesets.md:38]: Updated placeholder.

File: products/ruleset-engine/src/content/basic-operations/view-rulesets.md [hunk 4]
```diff
@@ -109,7 +105,7 @@ header: Response
       "phase": "http_request_firewall_custom"
     },
     {
-      "id": "{account-level-phase-ruleset-id}",
+      "id": "<ACCOUNT_PHASE_RULESET_ID>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/basic-operations/view-rulesets.md:108]: Updated placeholder.

File: products/ruleset-engine/src/content/custom-rulesets/deploy-custom-ruleset.md [hunk 5]
```diff
@@ -50,50 +49,50 @@ header: Response
...
         "description":"Execute custom ruleset",
         "action_parameters": {
-          "id": "{custom-ruleset-id}",
+          "id": "<CUSTOM_RULESET_ID>",
           "version": "latest"
         },
         "expression": "cf.zone.name == \"example.com\"",
         "last_updated": "2021-03-18T18:35:14.135697Z",
-        "ref": "{root-rule-id}",
+        "ref": "<PHASE_RULE_REF>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/custom-rulesets/deploy-custom-ruleset.md:69]: Fixed this placeholder.

File: products/ruleset-engine/src/content/custom-rulesets/deploy-custom-ruleset.md [hunk 6]
```diff
@@ -50,50 +49,50 @@ header: Response
...
         "enabled": true
       },
       {
-        "id": "{existing-phase-rule-id-1}",
+        "id": "<EXISTING_PHASE_RULE_ID_1>",
         "version": "1",
         "action": "execute",
         "action_parameters": {
-          "id": "{id-of-ruleset-executed-by-existing-rule-id}",
+          "id": "<EXECUTED_RULESET_ID_1>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/custom-rulesets/deploy-custom-ruleset.md:77]: Updated.

File: products/ruleset-engine/src/content/custom-rulesets/deploy-custom-ruleset.md [hunk 7]
```diff
@@ -50,50 +49,50 @@ header: Response
...
         "enabled": true
       },
       {
-        "id": "{existing-phase-rule-id-2}",
+        "id": "<EXISTING_PHASE_RULE_ID_2>",
         "version": "1",
         "action": "execute",
         "action_parameters": {
-          "id": "{id-of-ruleset-executed-by-existing-rule-id-2}",
+          "id": "<EXECUTED_RULESET_ID_2>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/custom-rulesets/deploy-custom-ruleset.md:90]: Updated.

File: products/ruleset-engine/src/content/custom-rulesets/deploy-custom-ruleset.md [hunk 8]
```diff
@@ -50,50 +49,50 @@ header: Response
...
           "version": "latest"
         },
         "expression": "cf.zone.name eq  \"example.com\"",
         "last_updated": "2021-03-16T15:50:29.861157Z",
-        "ref": "{existing-phase-rule-ref}",
+        "ref": "<EXISTING_PHASE_RULE_REF_2>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/custom-rulesets/deploy-custom-ruleset.md:95]: Updated.

File: products/ruleset-engine/src/content/managed-rulesets/deploy-managed-ruleset.md [hunk 9]
```diff
@@ -120,24 +118,24 @@ header: Response
 ---
 {
   "result": {
-    "id": "{zone-level-phase-ruleset-id}",
+    "id": "<ZONE_PHASE_RULESET_ID>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/managed-rulesets/deploy-managed-ruleset.md:121]: Updated.

File: products/ruleset-engine/src/content/managed-rulesets/deploy-managed-ruleset.md [hunk 10]
```diff
@@ -120,24 +118,24 @@ header: Response
...
         "action_parameters": {
-          "id": "{cloudflare-managed-ruleset-id}",
+          "id": "<CLOUDFLARE_MANAGED_RULESET_ID>",
           "version": "latest"
         },
         "expression": "true",
         "description": "Execute Cloudflare Managed Ruleset on my zone-level phase entry point",
         "last_updated": "2021-03-18T18:08:14.003361Z",
-        "ref": "{ruleset-ref-1}",
+        "ref": "<RULE_REF_1>",
```
Comment (pedrosousa) [inline products/ruleset-engine/src/content/managed-rulesets/deploy-managed-ruleset.md:138]: Fixed.

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**